backend/src/main/resources/liquibase/create-tables-changelog-1.0.xml

```diff
@@ -1238,4 +1238,86 @@
                                  referencedColumnNames="id"
                                  referencedTableName="keyvalue"/>
     </changeSet>
+
+    <changeSet author="lezhevg" id="new_lineage_id">
+        <createTable tableName="lineage_element">
+            <column name="id" type="int8" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_connection_entity">
+        <createTable tableName="lineage_connection">
+            <column name="element_id" type="int8">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="entity_id" type="int8">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="connection_type" type="int4">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="entity_type" type="int4">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_versioning_blob_entity">
+        <createTable tableName="lineage_versioning_blob">
+            <column name="id" type="int8" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="repository_id" type="int8">
+                <constraints nullable="true"/>
+            </column>
+            <column name="blob_sha" type="varchar(256)">
+                <constraints nullable="true"/>
+            </column>
+            <column name="blob_type" type="varchar(256)">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet author="lezhevg" id="new_lineage_experiment_run_entity">
+        <createTable tableName="lineage_experiment_run">
+            <column name="id" type="int8" autoIncrement="true">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="experiment_run_id" type="varchar(256)">
+                <constraints nullable="true"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="indexes-on-lineage" author="lezhevg">
+        <createIndex indexName="lineage_versioning_blob_index"
+          tableName="lineage_versioning_blob" unique="false">
+            <column name="repository_id" type="int8">
+            </column>
+            <column name="blob_sha" type="varchar(256)">
+            </column>
+            <column name="blob_type" type="varchar(256)">
+            </column>
+        </createIndex>
+        <createIndex indexName="lineage_experiment_run_index"
```

Comment:
Add indexes in separate changeset with the precondition  
```
<preConditions onFail="MARK_RAN">
    <not>
       <indexExists indexName="er_o"/>
    </not>
</preConditions>
```

Reply:
added